python/ray/_private/services.py

```diff
@@ -1445,9 +1445,8 @@ def get_address(redis_address):
 def start_gcs_server(
     redis_address: str,
     log_dir: str,
+    ray_log_sink_filename: Optional[str],
```

Comment:
nit: log_file?

IIUC, it's not just filename but the full path.

Reply:
Discussed offline, `ray_log_sink_filename` renamed to `ray_log_filepath`.